### PR TITLE
support argument CLAIR_OUTPUT for vulnerabilities output

### LINKED ISSUE #15: Allow Response to be limited by Severity Level
It seems that the Klar just outputs all vulnerabilities no matter what the severity level.  It would be nice to have an option for the `Klar` command to limit the response.  I do like the summary portion of it though.

Something like this:
`<ENV_VARS> klar --out_level="High" <image>`

The output would be something similar to this:
```
CVE-2016-1252: [High]
A man-in-the-middle attacker could circumvent the InRelease signature of a repository, leading to a malicious package being installed and, therefore, remote arbitrary code execution.
http://people.ubuntu.com/~ubuntu-security/cve/CVE-2016-1252
-----------------------------------------
Negligible: 10
Low: 69
Medium: 86
High: 1
```

### PR DESCRIPTION
**Pull request by supereagle**
fix #15

@hashmap  @hulu1522  PTAL.

### CODE REVIEW

**Comment by hulu1522** [comment]
@supereagle Are you sure you want to default the value to 'Unknown'?  Other then that it LG2M.  +1

File: main.go [hunk 1]
```diff
@@ -57,21 +77,30 @@ func main() {
...
+	iteratePriorities(priorities[0], func(sev string) { fmt.Printf("%s: %d\n", sev, len(store[sev])) })
 
 	if highSevNumber > threshold {
 		os.Exit(1)
 	}
 }
 
-func iteratePriorities(f func(sev string)) {
+func iteratePriorities(output string, f func(sev string)) {
+	filtered := true
```
Comment (hashmap) [inline main.go]: What if we just iterate over a subslice of `priorities` starting from the index of `output`? It may improve readability.

Comment (supereagle) [inline main.go]: Do you think the `filtered` is not enough readable? Not sure whether the following explain is acceptable.
In the case of iteration over a subslice of `priorities`, there will be start and end index. Both for priorities before the start index and after the end index, `filtered` is true as they will be filtered. Only for priorities between the start index and the end index, `filtered` is false as they will be outputted.

**Comment by supereagle** [comment]
@hulu1522  I prefer the default value as 'Unknown' for: 
* No effect for users who are not aware of this feature when update klar; 
* No sure the wanted default value for most users. Let the users to pass the value if they want this feature.

**Pull request merged**